gpu-operator/platform-support.rst

```diff
@@ -434,17 +434,16 @@ Support for KubeVirt and OpenShift Virtualization
 
 Red Hat OpenShift Virtualization is based on KubeVirt.
 
-================    ===========   =============   =========    =============    ========
+================    ===========   =============   =========    =============    ===========
 Operating System    Kubernetes           KubeVirt              OpenShift Virtualization
-----------------    -----------   -------------------------    -------------------------
+----------------    -----------   -------------------------    ----------------------------
 \                   \             | GPU           vGPU         | GPU            vGPU
                                   | Passthrough                | Passthrough
-================    ===========   =============   =========    =============    ========
-Ubuntu 20.04 LTS    1.22---1.28   0.36+           0.59.1+
-Ubuntu 22.04 LTS    1.22---1.28   0.36+           0.59.1+
-Red Hat Core OS                                                4.11---4.14      4.13,
-                                                                                4.14
-================    ===========   =============   =========    =============    ========
+================    ===========   =============   =========    =============    ===========
+Ubuntu 20.04 LTS    1.22---1.29   0.36+           0.59.1+
+Ubuntu 22.04 LTS    1.22---1.29   0.36+           0.59.1+
+Red Hat Core OS                                                4.11---4.15      4.13, 4.15
```

Comment:
@cdesiniotis, vGPU is 4.13 and 4.15, not through 4.15?  I don't know any details, I'm just double checking.

Reply:
This is a typo. It should be 4.13 through 4.15.

Reply:
I fixed this now.